db/migrate/20230718173344_create_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :email
+      t.string :photo
+
+      t.timestamps
+    end
```

Comment:
- [ ] Let's try to work with the fields given in the ERD for models. For users, we're to have a name, photo, bio, and post_counter fields. Kindly modify your migration to match the fields given in the ERD.
```ruby
    t.string :name
    t.string :photo 
    t.text :bio
    t.integer :posts_counter
```

<img width="260" alt="Screenshot 2023-07-18 at 6 40 10 PM" src="https://github.com/harshi0102/blog-app-rails/assets/50941074/af8b0a51-30ea-4e53-bbe6-16e256307300">